tests/startup_test.rs

```diff
@@ -11,7 +11,7 @@ fn start_serve() {
     assert_eq!(None, dura.get_runtime_lock());
 
     dura.start_async(&["serve"], true);
-    dura.wait();
+    dura.primary.as_ref().map(|d| d.read_line(8).unwrap());
```

Comment:
It's unclear here (and other occurrences) what the `8` refers to. Could the magic number be replaced with a `const` at the top of the module.

Reply:
fixed